lib/modules/hosts/wikipedia.js

```diff
@@ -6,10 +6,40 @@ export default {
 	name: 'wikipedia',
 	domains: ['wikipedia.org', 'wikipedia.com'],
 	logo: 'https://en.wikipedia.org/static/favicon/wikipedia.ico',
-	detect: ({ href }) => (/^https?:\/\/([A-z]{2}|simple)?\.(?:m\.)?wikipedia\.(?:com|org)\/wiki\/([^#\?]+)/i).exec(href),
-	async handleLink(href, [, country, article]) {
+	detect: url => {
+		return {
+			article: url.pathname.substr(6), // remove "/wiki/"
```

Comment:
Is it worth double-checking this is an article page before asking the wikipedia api?